versions.tf

```diff
@@ -1,3 +1,3 @@
 terraform {
-  required_version = "~> 0.11"
+  required_version = "~> 0.11, < 0.12"
```

Comment:
@rdunn-Hypr without this you are able to use 0.12 with the 0.11 version of the code. This results in lots of errors etc. I believe this limitation ensures people can't us 0.12 to run 0.11 which is a good safety measure.

Reply:
I did think about this the other day and wondered if the ~ would even as I believe it only works on constraining the major number. thanks for finding and testing this!